downstream/modules/platform/proc-modifying-the-run-schedule.adoc

```diff
@@ -0,0 +1,55 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-07-15
+:_mod-docs-content-type: PROCEDURE
+
+[id="modifying-the-run-schedule_{context}"]
+= Modifying the run schedule
+
+
+You can configure `metrics-utility` to run at specified times and intervals. Run frequency is expressed in cronjobs. See link:https://www.redhat.com/sysadmin/linux-cron-command[How to schedule jobs using the Linux ‘Cron’ utility] for more information. 
+
+== On RHEL
+.Procedure
+ 
+. From the command line, run: 
++
+[source, ]
+----
+crontab -e 
+----
++
+. After the code editor has opened, update the `gather` and `build` parameters using cron syntax as shown below: 
++
+[source, ]
+----
+*/2 * * * *     metrics-utility gather_automation_controller_billing_data --ship --until=10m
+*/5 * * * *     metrics-utili
+ty build_report
+----
++
+. Save and close the file.
+
+== On {OCPShort} from the {PlatformNameShort} operator 
+.Procedure
+
+. From the navigation panel, select *Workloads > Deployments*.
+. On the next screen, select *automation-controller-operator-controller-manager*.
+. Beneath the heading *Deployment Details*, click the down arrow button to change the number of pods to zero. This will pause the deployment so you can update the running schedule. 
+. From the navigation panel, select *Installed Operators*. From the list of installed operators, select {PlatformNameShort}. 
+. On the next screen, select the {ControllerName} tab. 
+. From the list that appears, select your {ControllerName} instance. 
+. On the next screen, select the `YAML` tab. 
+. In the `YAML` file, find the following parameters and enter a variable representing how often `metrics-utility` should gather data and how often it should produce a report: 
++
+[source, ]
+----
+metrics_utility_cronjob_gather_schedule:
+metrics_utility_cronjob_report_schedule: 
+----
++
+. Click btn:[Save].
+. From the navigation menu, select *Deployments* and then select *automation-controller-operator-controller-manager*.
```

Comment:
Suggest changing the menu selection to: select menu:Deployments[]

Reply:
updated